README.md

```diff
@@ -155,6 +155,30 @@ terraform:
   # ...
 ```
 
+You can also let tfnotify add a label to PRs whose `terraform plan` output result in no change to the current infrastructure.
```

Comment:
Could you mention about that this feature supports only GitHub now, here? Since tfnotify supports various platforms, I want to provide the same functionality as much as possible. But this time it’s only GitHub. Therefore it is user friendly to mention it here at least!

Reply:
Fair point! I added a sentence saying that it is Github notifications only.